src/containers/KeystoreUploadContainer/KeystoreUploadContainer.js

```diff
@@ -65,7 +65,7 @@ class KeystoreUploadContainer extends PureComponent {
             if (privateKey.length) {
               const address = importAddressFromPrivateKey(privateKey);
```

Comment:
`GTsqojGaG2sy4uUTwyqwjxDtaVaF9ja5DV`
Add temp address here instead of the following line.

Reply:
fixed